tensorflow/core/platform/posix/posix_file_system.cc

```diff
@@ -34,6 +37,8 @@ limitations under the License.
 
 namespace tensorflow {
 
+constexpr size_t kPosixCopyFileBufferSize = 1024 * 1024;
```

Comment:
128kB is the optimal according to this guy: https://eklitzke.org/efficient-file-copying-on-linux

Reply:
Updated.